docs/content/Getting-Started/Cloud/01-Overview.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: Overview
+permalink: /getting-started/cloud/overview
+category: Getting Started
+subCategory: Cube Cloud
+menuOrder: 2
+isDisableFeedbackBlock: true
+---
+
+Learn how to set up a new deployment on Cube Cloud using a sample database.
```

Comment:
This is a great example of a place where a short paragraph that explains "why should one do that" would be really cool to have.